playbooks/service-provision.yaml

```diff
@@ -94,6 +93,132 @@
                      | json_query("[?key!='guid' && key!='uuid' && key!='cloud_tags']")
                      | items2dict | combine({"cloud_tags": vars.cloud_tags}) }}
 
+  - name: Report ResourceClaim
+    debug:
+      msg: "babylon.resourceClaim: {{ user_namespace }} {{ resource_claim_name }}"
+
+  - block:
+    - name: Wait for ResourceHandle to be assigned to ResourceClaim {{ resource_claim_name }}
+      k8s_info:
+        kubeconfig: "{{ kubeconfig }}"
+        api_version: poolboy.gpte.redhat.com/v1
+        kind: ResourceClaim
+        namespace: "{{ user_namespace }}"
+        name: "{{ resource_claim_name }}"
+      register: r_claim
+      retries: 30
+      delay: 2
+      until: >-
+        'resourceHandle' in r_claim.resources[0].status
```

Comment:
Is there always at least one resource and it has always a status key ?

In other words do you need to add `default()` ?:

```'resourceHandle' in r_claim.resources[0].status | default({})```